src/omero_zarr/masks.py

```diff
@@ -404,10 +404,11 @@ def masks_to_labels(
         properties: Dict[int, Dict] = {}
 
         for count, shapes in enumerate(masks):
-            mask_value = count + 1  # Prevent zeroing
-            # All shapes same color for each ROI
             for mask in shapes:
-                # Unused metadata: the{ZTC}, x, y, width, height, textValue
+                # Using ROI ID allows stitching label from multiple images
+                # into a Plate and not creating duplicates from different iamges.
+                # All shapes will be the same value (color) for each ROI
```

Comment:
This sounds correct since we only support a `UNION`  interpretation of shapes within a single ROI.